axelrod/tests/unit/test_fingerprint.py

```diff
@@ -119,6 +232,19 @@ def test_fingerprint_with_filename(self):
             data = out.read()
             self.assertEqual(len(data.split("\n")), 10)
 
+    def test_fingerprint_with_filename_none_and_in_memory_false(self):
```

Comment:
this test is new.  it just failed (since i use windows, it had different behavior on my machine) .